eslint.config.js

```diff
@@ -0,0 +1,45 @@
+import globals from 'globals'
+import vueParser from 'vue-eslint-parser'
+import eslint from '@eslint/js'
+import tsESLint from 'typescript-eslint'
+import eslintPluginVue from 'eslint-plugin-vue'
+
+const commonLint = {
+  languageOptions: {
+    globals: {
+      ...globals.browser
+    }
+  }
+}
+
+const vueLint = {
+  files: ['*.vue', '**/*.vue'],
+  languageOptions: {
+    parser: vueParser,
```

Comment:
```suggestion
    "parser": "vue-eslint-parser"
```

公式サイトだとこういう風になっているサイトしか見つからなかったので、こうしておくのが無難かなと思います。
https://eslint.vuejs.org/user-guide/#how-to-use-a-custom-parser

Reply:
あれ、これその後のコミットで直しております

Reply:
あ、ほんとだ